### PR TITLE
Qt/Debugger: Add Ctrl+F (find thread), Ctrl+C (copy)

### PR DESCRIPTION
**Pull request by elad335**
* Ctrl+F: focues on the combo box which allows to search thread by text.
* Ctrl+C (handled by QListWidget): copies selected instruction contents. 
* Propagate unhandled key presses to the main window.

### CODE REVIEW

File: rpcs3/rpcs3qt/debugger_frame.cpp [hunk 1]
```diff
@@ -212,6 +213,15 @@ bool debugger_frame::eventFilter(QObject* object, QEvent* event)
...
 		keyPressEvent(static_cast<QKeyEvent*>(event));
 	}
+
+	if (object == m_choice_units && event->type() == QEvent::FocusOut)
+	{
+		if (auto index = m_choice_units->currentIndex(); index >= 0)
+		{
+			m_choice_units->setCurrentText(m_choice_units->itemText(index));
+		}
+	}
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/debugger_frame.cpp:223]: missing comment/explanation why this is needed.

Comment (elad335) [reply rpcs3/rpcs3qt/debugger_frame.cpp]: Added.

Comment (Megamouse) [reply rpcs3/rpcs3qt/debugger_frame.cpp]: isn't that the same as m_choice_units->clearEditText();
?

Comment (elad335) [reply rpcs3/rpcs3qt/debugger_frame.cpp]: That just makes the textbox empty with placeholder text.

File: rpcs3/rpcs3qt/debugger_frame.cpp [hunk 2]
```diff
@@ -207,11 +208,25 @@ void debugger_frame::ChangeColors() const
...
 {
-	// There's no overlap between keys so returning true wouldn't matter.
 	if (object == m_debugger_list && event->type() == QEvent::KeyPress)
 	{
 		keyPressEvent(static_cast<QKeyEvent*>(event));
+
+		const bool handled = event->isAccepted();
+		event->accept(); // Restore accepted state
+		return handled;
+
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/debugger_frame.cpp:218]: empty line

File: rpcs3/rpcs3qt/debugger_frame.cpp [hunk 3]
```diff
@@ -207,11 +208,25 @@ void debugger_frame::ChangeColors() const
...
 bool debugger_frame::eventFilter(QObject* object, QEvent* event)
 {
-	// There's no overlap between keys so returning true wouldn't matter.
 	if (object == m_debugger_list && event->type() == QEvent::KeyPress)
 	{
 		keyPressEvent(static_cast<QKeyEvent*>(event));
+
+		const bool handled = event->isAccepted();
+		event->accept(); // Restore accepted state
+		return handled;
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/debugger_frame.cpp:217]: why is this needed

Comment (elad335) [reply rpcs3/rpcs3qt/debugger_frame.cpp:217]: I guess it is not really needed.

Comment (elad335) [reply rpcs3/rpcs3qt/debugger_frame.cpp]: Removed.

File: rpcs3/rpcs3qt/debugger_frame.cpp [hunk 4]
```diff
@@ -211,7 +212,18 @@ bool debugger_frame::eventFilter(QObject* object, QEvent* event)
 	if (object == m_debugger_list && event->type() == QEvent::KeyPress)
 	{
 		keyPressEvent(static_cast<QKeyEvent*>(event));
+		event->accept(); // Restore accepted state
 	}
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/debugger_frame.cpp:216]: missing return

Comment (elad335) [reply rpcs3/rpcs3qt/debugger_frame.cpp:217]: Fixed,
